src/sentry/static/sentry/app/views/organizationEventsV2/tagsTable.jsx

```diff
@@ -13,7 +20,11 @@ const TagsTable = props => {
           {props.tags.map(tag => (
             <StyledTr key={tag.key}>
               <TagKey>{tag.key}</TagKey>
-              <TagValue>{tag.value}</TagValue>
+              <TagValue>
+                <a href="#" onClick={handleSelect(tag)}>
```

Comment:
It would be nice if this had an actual URL so you can, e.g. open in a new tab. Otherwise can we use a button to be a bit more semantic?

Reply:
I could generate URLs by snaking the `location` down through props.